kedro/config/omegaconf_config.py

```diff
@@ -129,6 +129,8 @@ def __init__(
             env=env,
             runtime_params=runtime_params,
         )
+        if not self.runtime_params:
+            self.runtime_params = {}
```

Comment:
Probably nicer to do `runtime_params=runtime_params or {}` above instead?

Reply:
Or even better `AbstractConfigLoader` should actually be the class responsible for this defaulting behaviour, i.e. it should do `self.runtime_params = runtime_params or {}`.

Reply:
100% agree with this.